hedera-node/src/main/java/com/hedera/services/txns/consensus/TopicUpdateTransitionLogic.java

```diff
@@ -63,7 +63,7 @@
 public class TopicUpdateTransitionLogic implements TransitionLogic {
 	private static final Logger log = LogManager.getLogger(TopicUpdateTransitionLogic.class);
 
-	private final Function<TransactionBody, ResponseCodeEnum> PRE_SIGNATURE_VALIDATION_SEMANTIC_CHECK =
+	private final Function<TransactionBody, ResponseCodeEnum> preSignatureValidationSemanticCheck =
 			this::validatePreSignatureValidation;
```

Comment:
Remove this, return directly `this::validatePreSignatureValidation` in `semanticCheck`